demisto_sdk/commands/lint/linter.py

```diff
@@ -1306,3 +1337,280 @@ def _get_commands_list(self, script_obj: dict):
         except Exception:
             logger.debug("Failed getting the commands from the yml file")
         return commands_list
+
+    class DockerImageFlagOption(Enum):
+        FROM_YML = "from-yml"
+        NATIVE = "native:"
+        NATIVE_LATEST = "native:latest"
+        ALL_IMAGES = "all"
+
+    def _get_latest_native_image(
+        self, script_id: str, supported_native_images: Set[str]
+    ) -> Union[str, None]:
+        """
+        Gets the latest tag of the native image from Docker Hub.
+        If the integration/script that lint runs on doesn't support the latest native image - write to the logs that the
+        checks on docker will be skipped.
+        Args:
+            script_id (str): The ID of the integration/script that lint runs on.
+            supported_native_images (set): A set including the names of the native images supported for the
+                                        script/integration that lint runs on.
+        Returns: The latest native image.
+        """
+        log_prompt = f"{self._pack_name} - Get Latest Native Image"
+        logger.info(f"{log_prompt} - Started")
+
+        if (
+            self.DockerImageFlagOption.NATIVE_LATEST.value
```

Comment:
we make this same check for versioned images. Lets make this check once for both before these functions are called

Reply:
Implemented as a function - '_is_native_image_supported' which is called once.